ckan/views/dataset.py

```diff
@@ -1108,21 +1122,78 @@ def activity(package_type: str, id: str) -> Union[Response, str]:  # noqa
         u'auth_user_obj': g.userobj
     })
     data_dict = {u'id': id}
+    base_limit = int(config.get(u'ckan.activity_list_limit', 30))
+    max_limit = int(config.get(u'ckan.activity_list_limit_max', 100))
```

Comment:
Since https://github.com/ckan/ckan/pull/6467, if you have declared a config option it will automatically get validated and parsed (and the defaults used) if you use `config.get_value()`

```suggestion
    base_limit = config.get_value('ckan.activity_list_limit')
    max_limit = config.get_value('ckan.activity_list_limit_max')
```